json/src/lib.rs

```diff
@@ -1124,6 +1124,39 @@ pub struct SignRawTransactionInput {
     pub amount: Option<Amount>,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct GetNetTotalsResult {
+    /// Total bytes received
+    #[serde(rename = "totalbytesrecv")]
+    pub total_bytes_recv: u64,
+    /// Total bytes sent
+    #[serde(rename = "totalbytessent")]
+    pub total_bytes_sent: u64,
+    /// Current UNIX time in milliseconds
+    #[serde(rename = "timemillis")]
+    pub time_millis: u64,
+    /// Upload target statistics
+    #[serde(rename = "uploadtarget")]
+    pub upload_target: GetNetTotalsResultUploadTarget,
+}
+
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct GetNetTotalsResultUploadTarget {
+    /// Length of the measuring timeframe in seconds
+    #[serde(rename = "timeframe")]
+    pub time_frame: u64,
```

Comment:
Sounds like this should be a `Duration` except if we handle time the same way elsewhere.

Reply:
Ah, I see there's a TODO comment about that but time is still handle as timestamps and seconds. Maybe we should discuss this at some point.